lib/api/route/streams.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+part 'streams.g.dart';
```

Comment:
Let's name this file `channels.dart` instead of `streams.dart`, so you don't have to rename it later as part of #631 :slightly_smiling_face: